terra/task.py

```diff
@@ -145,10 +136,9 @@ def __call__(self, *args, **kwargs):
         return_value = self.run(*args, **kwargs)
       finally:
         settings.terra.zone = original_zone
-    self.settings = None
     return return_value
 
-  # from https://stackoverflow.com/a/45333231/1771778
-  def on_failure(self, exc, task_id, args, kwargs, einfo):
-    logger.exception('Celery task failure!!!1', exc_info=exc)
-    super(TerraTask, self).on_failure(exc, task_id, args, kwargs, einfo)
+  # # from https://stackoverflow.com/a/45333231/1771778
```

Comment:
Does this not do what I thought it did?

Reply:
It does, I'm just not 100% ready to include this, until I have a "demo" app to see how it all plays through.